src/protocols/bdx/BDXMessageUtils.cpp

```diff
@@ -424,16 +375,29 @@ CHIP_ERROR ReceiveAccept::Parse(System::PacketBuffer * aBuffer, ReceiveAccept &
     }
 
     // Rest of message is metadata (could be empty)
-    if (bufReader.OctetsRead() < aBuffer->DataLength())
+    if (bufReader.OctetsRead() < aBuffer.DataLength())
```

Comment:
I feel like this is another case where `bufReader.Remaining() > 0` would be much easier to understand.

Reply:
Ack